src/Microsoft.ML.DataView/VectorType.cs

```diff
@@ -21,6 +22,16 @@ public sealed class VectorDataViewType : StructuredDataViewType
         /// As with <see cref="Size"/>, a zero value indicates that the vector type is considered to have
         /// unknown length along that dimension.
         /// </summary>
+        /// <remarks>
+        /// In the case where this is a multi-dimensional type, that is, a situation where <see cref="Dimensions"/>
+        /// has length greater than one, since <see cref="VBuffer{T}"/> itself is a single dimensional structure,
+        /// we must clarify what we mean. The indices represent a "flattened" view of the coordinates implicit in the
+        /// dimensions. We consider that that last dimension is the most "minor" index. In the case where <see cref="Dimensions"/>
```

Comment:
>that [](start = 41, length = 4)

maybe remove one that?

Reply:
"that that last dimension" is more grammatically correct I think but I can change the words since I agree the phrasing is awkward. That *the* last dimension?